Bottom: 2e26dc059816e7c7a59e02052cd8689d982a66a4
Top:    6023063f46a44d361762a6721e77c5b797a0a751
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-04 03:01:26 +0100

Refresh of hid-gtk-create-a-gtktoolbar-of

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 27c6938..04bfeb6 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -105,8 +105,8 @@ typedef struct
 
   GtkWidget *top_hbox,
     *menu_hbox, *position_hbox,
-    *mode_buttons_vbox, *menubar_toolbar_vbox,
-    *mode_buttons_frame, *mode_buttons_frame_vbox;
+    *menubar_toolbar_vbox,
+    *mode_buttons_frame;
   GtkWidget *left_toolbar;
   GtkWidget *menu_bar, *layer_selector;
   GtkWidget *mode_toolbar;
